Shuttlerock/actions
# Create a pull request when development begins on a Jira issue

## Create a pull request when development begins on a Jira issue

[Clubhouse Story](https://app.clubhouse.io/shuttlerock/story/75383)



## How to test the PR

1. Close any open PRs for `[STUDIO236]` [here](https://github.com/Shuttlerock/actions/pulls)
1. Delete any branches named `studio-236-create-a-release-pr`.
1. Move [this issue](https://shuttlerock.atlassian.net/browse/STUDIO-236) from `Ready for Development` to `In Development` in Jira.
1. It should create a PR and send me (@dave) a slack message.

## Deployment Notes

None
